tests/Tests/600__ai_apps/600__anaconda_commercial_edition/600__anaconda_commercial_edition.robot

```diff
@@ -46,6 +52,24 @@ Verify Anaconda Commercial Edition Fails Activation When Key Is Invalid
   Capture Page Screenshot  enabletab_anaconda_notpresent.png
   Page Should Not Contain Element  xpath://div[@class="pf-c-card__title"]/span[.="Anaconda Commercial Edition"]
 
+Verify User Is Able to Activate Anaconda Commercial Edition
+  [Tags]  Sanity  Smoke
+  ...     ODS-272  ODS-344  ODS-501
+  Open Browser  ${ODH_DASHBOARD_URL}  browser=${BROWSER.NAME}  options=${BROWSER.OPTIONS}
+  Login To RHODS Dashboard  ${TEST_USER.USERNAME}  ${TEST_USER.PASSWORD}  ${TEST_USER.AUTH_TYPE}
+  Wait for RHODS Dashboard to Load
+  Enable Anaconda  ${ANACONDA_CE.ACTIVATION_KEY}
+  Wait Until Keyword Succeeds    50  1  Page Should Not Contain Element    xpath://*/div[contains(@class, "bullseye")]
+  Capture Page Screenshot  anaconda_success_activation.png
+  Menu.Navigate To Page    Applications    Enabled
+  Wait Until RHODS Dashboard JupyterHub Is Visible
+  Capture Page Screenshot  enabletab_anaconda_present.png
+  Page Should Contain Element  xpath://div[@class="pf-c-card__title"]/span[.="Anaconda Commercial Edition"]
+  ${val_result}=  Get Anaconda Validator Logs
+  Log  ${val_result}
+  Should Be Equal  ${val_result}  ${val_success_msg}
+  Wait Until Keyword Succeeds    1200  1  Check Anaconda CE Image Build Status  Complete
+  Spawn and Check JupyterLab Using Anaconda CE Image
 
 ** Keywords ***
```

Comment:
I think these keywords should be moved to one of the general files (perhaps JupyterLabLauncher) rather than staying in this Test file.

Reply:
Not totally agree with this because a part from the one in your next comment, I think the keywords here are only meaningful for Anaconda TC. 
But, I would agree in case we decide to create a Resource object for AI_Apps and put here all the keywords used by this component. I could start doing from the next TC and as soon as we plan a refactor, we insert this among the task.

What do you think?

Reply:
@lugi0 I've done a bit of refactoring abstracting some keywords and moving out some of them in Resource folder.